docs/configuration/configuration.rst

```diff
@@ -2,5 +2,168 @@
+are calculated using an estimator (this is described in ??????).
+
+TARDIS currently supports three different kinds of line interaction: ``scatter`` - a resonance scattering implementation,
+``macroatom`` - the most complex form of line interaction described in :ref:`macroatom` and ``downbranch`` a simplified
+version of ``macroatom`` in which only downward transitions are allowed.
+
+Finally, ``w_epsilon`` describes the dilution factor to use to calculate :math:`J_\textrm{blue}` that are 0, which
+causes problem with the code (so :math:`J_\textrm{blue}` are set to a very small number).
+
+NLTE
+^^^^
+
+.. code-block:: yaml
+
+    nlte:
+        coronal_approximation: True
+        classical_nebular: False
+
+The NLTE configuration currently allows setting ``coronal_approximation`` which sets all :math:`J_\textrm{blue}` to 0.
+This is useful for debugging with :term:`chianti` for example. Furthermore one can enable 'classical_nebular' to set all
+:math:`\beta_\textrm{Sobolev}` to 1. Both options are used for checking with other codes and should not be enabled in
+normal operations.
+
+
+
+Monte Carlo
+-----------
+
+The ``montecarlo`` section describes the parameters for the MonteCarlo radiation transport and convergence criteria:
+
+.. jsonschema:: schemas/montecarlo.yml
+
+The ``seed`` parameter seeds the random number generator first for the creation of the packets
+(:math:`\nu` and :math:`\mu`) and then the interactions in the actual MonteCarlo process.
+The ``no_of_packets`` parameter can take a float number for input convenience and gives the number of packets normally
+used in each MonteCarlo loop. The parameters ``last_no_of_packets`` and ``no_of_virtual_packets`` influence the last run
+of the MonteCarlo loop when the radiation field should have converged. ``last_no_of_packets`` is normally higher than
+``no_of_packets`` to create a less noisy output spectrum. ``no_of_virtual_packets`` can also be set to greater than 0 to
+use the Virtual Packet formalism (reference missing ?????). The ``iterations`` parameter describes the maximum number of
+MonteCarlo loops executed in a simulation before it ends. Convergence criteria can be used to make the simulation stop
+sooner when the convergence threshold has been reached.
+
+The ``convergence_criteria`` section again has a ``type`` keyword. Two types are allowed: ``damped`` and ``specific``.
+All convergence criteria can be specified separately for the three variables for which convergence can be checked
+(``t_inner``, ``t_rad``, ``ws``) by specifying subsections in the ``convergence_criteria`` of the same name. These
+override then the defaults.
+
+
+#. ``damped`` only has one parameter ``damping-constant`` and does not check for convergence.
+
+#. ``specific`` checks for the convergence threshold specified in ``threshold``. For ``t_rad`` and ``w`` only a given
+    fraction (specified in ``fraction``) has to cross the ``threshold``. Once a convergence  threshold is read, the simulation
```

Comment:
Deprecated

Reply:
not sure what you mean - check the new version.